CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
   * Make `shippingMethod` property on `BTThreeDSecureRequest` an enum instead of a string
   * Remove `BTTokenizationService`
   * Make `BTPaymentMethodNonceParser` private
+  * Remove `BTAppSwitchDelegate`
```

Comment:
This delegate was something merchants could use to be notified when the SDK was about to open an `SFSafariViewController` (for 3DS and local payments) or switch to the Venmo app. We don't think this is necessary at this point. 

For the `SFSafariViewController`, the merchant is responsible for presenting it in their `BTViewControllerPresentingDelegate` methods, so they already know that it's being presented.

And for switching to the Venmo app, it's more reliable to use app delegate methods to detect when their app is transitioning from foreground to background and vice versa.